app/ViewStores.tsx

```diff
@@ -1,33 +1,40 @@
 import { ReactElement, useEffect, useState } from "react";
 import { Wrapper } from "./ui/Wrapper";
 import { styled } from "styled-components";
-import { Location } from "./types";
-import { darkGrey, black, greyInactiveButton } from "./ui/colourLibrary";
+import { Store } from "./types";
+import {
+  darkGrey,
+  black,
+  greyInactiveButton,
+  skyBlue,
+} from "./ui/colourLibrary";
 import { LargeButton } from "./ui/LargeButton";
 import { getStores } from "@/server/getStores";
+import { UpdateStore } from "./UpdateStore";
 
-export function ViewStores(props: {
-  onCreateClick: () => void;
-  onStoreClick: () => void;
-}): ReactElement {
-  const { onCreateClick, onStoreClick } = props;
-  const [stores, setStores] = useState<Location[]>([]);
+export function ViewStores(props: { onCreateClick: () => void }): ReactElement {
+  const { onCreateClick } = props;
+  const [stores, setStores] = useState<Store[]>([]);
+  const [storeIsSelected, setStoreIsSelected] = useState(false);
+  const [storeDetails, setPrefilledStoreDetails] = useState<Store>();
```

Comment:
I feel like we could achieve the same outcome with just the second state variable

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/20/commits/b400fc29a94335ab48e15377b2e2801895b05c6c

Reply:
let's remove the dummy store as discussed

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/20/commits/661c21886f6fb045c1ba9172806f04f0c7ccc34a